lib/postprocessing/image.js

```diff
@@ -87,23 +85,49 @@ function hasExtraInstructions(instructions) {
 }
 
 async function getImageMetadata(file) {
-    const start = new Date();
     try {
-        // [1x1+0+0] is a trick to avoid reading all pixels into memory, improving time for large images
-        const result = await gm(`${file}[1x1+0+0]`).write("json:-");
-        const json = JSON.parse(result.toString());
+        // suffix attributes with `#` to disable human-readable parsing if exiftool not used with `-n` flag
+        const attributes = [
+            "-Orientation",
+            "-FileType",
+            "-ImageHeight",
+            "-ImageWidth",
+            "-JPEGQualityEstimate"
+        ];
+        const attributesToExtract = attributes.join(" ");
+        const command = `exiftool -n -json ${attributesToExtract} ${file}`;
+        console.log(`Metadata (exiftool) command is: ${command}`);
+
+        const { stdout, stderr } = await exec(command);
+
+        if(stderr){
+            throw new GenericError(`Reading metadata from intermediate rendition failed: ${stderr}`);
+        }
 
-        // newer imagemagick returns an array (for multi-page formats), older just one object (6.8.9-9 at least)
-        if (Array.isArray(json)) {
-            return json[0].image;
-        } else {
-            return json.image;
+        let metadata = JSON.parse(stdout);
```

Comment:
The value returned is a string, not a parsed json object.

Reply:
Good catch!